scripts/release.ts

```diff
@@ -0,0 +1,44 @@
+import { exec, OutputMode } from "https://deno.land/x/exec@0.0.5/mod.ts";
+import { increment } from "https://deno.land/std@0.167.0/semver/mod.ts";
+import { Select } from "https://deno.land/x/cliffy@v0.25.5/prompt/mod.ts";
+
+await exec("git fetch --tags");
+
+const response = await exec(`bash -c "git tag | sort -V | tail -1"`, {
+  output: OutputMode.Capture,
+});
```

Comment:
This would break on windows, wouldn't it?

Reply:
Can you try?

Reply:
Sure!